### ISSUE TITLE
Canvas image compression when sending to ConfyUI (ver. 1.9.3)

### ISSUE THREAD

**Issue by DmitryLeek** [NimaNzrii/comfyui-photoshop#247]
Hello, I am very grateful to you for creating such a useful tool for my work.

However, I found a rather serious problem with the quality of the image sent from the plugin to ConfyUI. I found it in the process of changing the background, when the parts of the image that should have been left unchanged acquired strong distortions typical for jpg compression. 

To make sure of the problem I performed a simple experiment:

I generated an image and added it to the Photoshop canvas, then sent it for regeneration. In the ConfyUI scheme I added an image saving node to the Photoshop plugin node. Then I compared the two images and made sure that when the plugin takes the content of the canvas from Photoshop, it applies a strong compression to it, which creates artifacts. 

Screenshots and files are attached.

Thank you very much in advance for your help!

Just in case, I am attaching a link to a google drive with images in png format, that way you can open my workflow with you:
[https://drive.google.com/drive/folders/1XIQlCLXZJdpNQjWfgMTA90xaf0BQy4YN?usp=drive_link](url)

![Saved image from a plugin node](https://github.com/user-attachments/assets/2c43c82b-00c1-4b1b-a7ab-07fbe2a8d73f)
![The original image after generation](https://github.com/user-attachments/assets/188ce102-6af7-4c85-841f-7e5cce3f369c)

![Screenshot_2](https://github.com/user-attachments/assets/ee233cda-564d-496c-999e-6a62a76c4b94)
![Screenshot_3](https://github.com/user-attachments/assets/8e0a31a7-c9ac-4b51-b05b-fca7c8a171c9)

**Comment by LiJT** [comment]
Yeah, we have reported this problem already, We really need Nima back to address this prblem for professional use..
https://github.com/NimaNzrii/comfyui-photoshop/issues/261

**Comment by DmitryLeek** [comment]
> Yeah, we have reported this problem already, We really need Nima back to address this prblem for professional use.. #261

Yes, the problem is very serious, essentially screwing up all the work for the plugin

**Comment by MinGoo666** [comment]
而且限制了尺寸为1024？即使我放在PS里的文件大于1024，这就导致没法在原图上进行重绘等操作，出来的图会非常模糊
